src/gasnow.rs

```diff
@@ -25,24 +25,41 @@ struct CachedResponse {
 }
 
 #[derive(Clone, Copy, Debug, Default, serde::Deserialize, PartialEq)]
-struct Response {
-    code: u32,
-    data: ResponseData,
+pub struct Response {
+    pub code: u32,
+    pub data: ResponseData,
 }
 
 // gas prices in wei
 #[derive(Clone, Copy, Debug, Default, serde::Deserialize, PartialEq)]
-struct ResponseData {
-    rapid: f64,
-    fast: f64,
-    standard: f64,
-    slow: f64,
+pub struct ResponseData {
+    pub rapid: f64,
+    pub fast: f64,
+    pub standard: f64,
+    pub slow: f64,
 }
 
-const RAPID: Duration = Duration::from_secs(15);
-const FAST: Duration = Duration::from_secs(60);
-const STANDARD: Duration = Duration::from_secs(300);
-const SLOW: Duration = Duration::from_secs(600);
+pub const RAPID: Duration = Duration::from_secs(15);
+pub const FAST: Duration = Duration::from_secs(60);
+pub const STANDARD: Duration = Duration::from_secs(300);
+pub const SLOW: Duration = Duration::from_secs(600);
```

Comment:
Are these validity periods? 15, 60, 300, 600?

Reply:
Expected time it takes for a transaction to be mined with this gas price according to GasNow documentation.